README.md

```diff
@@ -140,6 +140,23 @@ func (c *quux) Echo(s string) string { return s }
 ```
 Note the type of `deps` parameter passed to `NewClient` constructor function is generated by Zanzibar, as indicated by the import path. Zanzibar takes care of initializing and passing in the acutal `deps` argument, as mentioned in [Dependency Injection](#dependency-injection).
 
+###### Circuit Breaker
+For increasing overall system resiliency, zanzibar uses a [Circuit Breaker](https://msdn.microsoft.com/en-us/library/dn589784.aspx) to avoid calling clients when there is a increase in failure rate beyond a set 
+threshold. It recovers it after elasped time to again fo in half-open and close state. 
```

Comment:
> It recovers it after elasped time to again fo in half-open and close state.

Can you rework this sentence? a few typos and the information it tries to convey doesn't seem clear to me.